### DISCUSSION TITLE
LocalAI won't open in Windows

### DISCUSSION THREAD

**Discussion by sm18lr88** [louisgv/local.ai#77] [Q&A] [answered]
I have Win 11 Pro with Python with standard libraries precompiled, Docker, CMake, MinGW, MongoDB, VSCode everything: Win SDKs, all C++ stuff, Cuda Toolkit 12.1, and Zlib.
Ryzen 5 4500U, 16 gb ram, SSD, no dedicated GPU.

I installed using the latest .exe and .msi installers, and when I click to open, nothing happens. Any clues?

**Comment by louisgv** [comment]
@sm18lr88 are you using the v0.5.2 version? - I thought it worked since we removed all the GPU related stuffs :d.... Ref: #62

**Reply by louisgv** [reply]
I also tested v0.5.2 in a real windows 11 machine as well, and it worked on my end

**Reply by louisgv** [reply]
Might be related to #61 - I'm pretty sure 0.5.2 works - 0.5.1 requires CUDA added to your path if you happen to grabbed that version

**Reply by sm18lr88** [reply]
I tried it again and nothing. But I tested it in a windows sandbox and it ran just fine.
Does this app rely on MS Edge? A quick search through the codebase suggests so. Let me know, because I obliterated any almost all trace of MS Edge from my pc.

**Reply by louisgv** [reply, marked as answer]
Ooh that's right - it relies on webview2 engine, which on Windows is based on the Edge runtime. One way to overcome this is to build a custom version of the app with webview bundled (note this will bloat the output by an additional 70MB), see: https://tauri.app/v1/guides/building/windows#webview2-installation-options

**Reply by sm18lr88** [reply]
That did the trick, thanks. Currently downloading a model, will test soon. 
Question, how does GPT4ALL list some of these models with much less RAM requirement?

**Reply by louisgv** [reply]
Oh the RAM req I added is purely based on my experimentation (for optimal usage), but please send PR to fix them if you think they're too conservative :p...
